src/entity/Entity.php

```diff
@@ -1080,6 +1080,13 @@ public function onCollideWithPlayer(Player $player) : void{
 
 	}
 
+	/**
+	 * Called when interacted or tapped by a Player
```

Comment:
Please also document the return value (an opaque true/false return has very unclear usage).